Fordítások/UltraCosmetics/config.yml

```diff
@@ -86,169 +86,169 @@ Economy: Vault
 # third-BLOCKS: BLOCKS "around around" the center.
 # below-chests: Block below the chests.
 # barriers: the barrier.
-# chest-type: The chest type (NORMAL or ENDER)
+# chest-Típus: The chest Típus (NORMAL or ENDER)
 # effect: The effect (complete list: http://pastebin.com/CVKkufck)
 # 
 # For permission command:
 # Change it to correspond to your Permissions plugin!
 # Default one is Pex's.
 # 
-TreasureChests:
-  Enabled: false
-  Key-Price: 1000
-  Loots:
-    Money:
-      Enabled: true
-      Max: 100
-      Chance: 10
-      Message:
-        enabled: false
-        message: '%prefix% &6&l%name% found %money%$'
-    Gadgets-Ammo:
-      Enabled: true
-      Min: 20
-      Max: 100
-      Chance: 60
-      Message:
-        enabled: false
-        message: '%prefix% &6&l%name% found %ammo% %gadget% ammo'
-    Gadgets:
-      Enabled: true
-      Chance: 20
-      Message:
-        enabled: false
-        message: '%prefix% &6&l%name% found gadget %gadget%'
-    Mounts:
-      Enabled: true
-      Chance: 10
-      Message:
-        enabled: false
-        message: '%prefix% &6&l%name% found rare %mount%'
-    Pets:
-      Enabled: true
-      Chance: 10
-      Message:
-        enabled: false
-        message: '%prefix% &6&l%name% found rare %pet%'
-    Morphs:
-      Enabled: true
-      Chance: 4
-      Message:
-        enabled: true
-        message: '%prefix% &6&l%name% found legendary %morph%'
-    Effects:
-      Enabled: true
-      Chance: 4
-      Message:
-        enabled: true
-        message: '%prefix% &6&l%name% found legendary %effect%'
-    Hats:
-      Enabled: true
-      Chance: 10
-      Message:
-        enabled: false
-        message: '%prefix% &6&l%name% found rare %hat%'
-    Suits:
-      Enabled: true
-      Chance: 5
-      Message:
-        enabled: false
-        message: '%prefix% &6&l%name% found rare %suit%'
```

Comment:
Ezeket kell pl.: lefordítani:

message: '%prefix% &6&l%name% talált egy ritka %suit%'

Reply:
De ezek le vannak fordítva

Reply:
Az rendben is volt. De ha jól láttam csak ezeket kell benne fordítani.